src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,231 @@
+import React, { forwardRef, useRef } from "react";
+import PropTypes from "prop-types";
+import { COLORS_BASIC, SIZES_BASIC } from "../../constants";
+import useMergeRefs from "../../hooks/useMergeRefs";
+import "./Slider.scss";
+import { bem, ensureDefaultValue } from "./SliderHelpers";
+import { SliderProvider } from "./SliderContext";
+import SliderBase from "./SliderBase/SliderBase";
+import SliderInfix from "./SliderInfix";
+
+const Slider = forwardRef(
+  (
+    {
+      ariaLabel,
+      ariaLabelledby,
+      className,
+      color,
+      "data-testid": dataTestId,
+      disabled,
+      id,
+      max,
+      min,
+      onChange,
+      ranged,
+      step,
+      showValue,
+      size,
+      value,
+      defaultValue,
+      valueFormatter,
+      valueText,
+      // ------ Additional subcomponents' props
+      indicateSelection,
+      prefix,
+      postfix
+    },
+    ref
+  ) => {
+    const componentRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+    const providerProps = {
```

Comment:
@niksa-monday IMO it is less clear creating an object and then spreading it on the Element.

I think it is clearer then just adding it to the component

Reply:
Changing ...

Reply:
Done!